### PR TITLE
add requests-html

### PR DESCRIPTION
**Pull request by ok-gitr**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ok-gitr** [comment]
@conda-forge/help-python please review and merge. thanks.

File: recipes/requests-html/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - pyquery
+    - python >=3.6
+    - requests
+    - w3lib
+
+test:
+  imports:
+    - requests_html
+  commands:
+    #- pip check #requests-html 0.10.0 requires bs4, which is not installed.
```
Comment (chrisburr) [inline recipes/requests-html/meta.yaml:36]: It should be installed as it's in `run`?

Comment (ok-gitr) [reply recipes/requests-html/meta.yaml:36]: yes, it should and it looks like it does but it still produces that error for some reason.

Comment (chrisburr) [reply recipes/requests-html/meta.yaml:36]: I don't see a failing commit, can you push one so the logs can be checked

Comment (ok-gitr) [reply recipes/requests-html/meta.yaml]: ok, i just edited and pushed

Comment (ok-gitr) [reply recipes/requests-html/meta.yaml]: @chrisburr the log shows the [error now](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=342776&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=1189)

Comment (chrisburr) [reply recipes/requests-html/meta.yaml]: It's a bug in the package, `bs4` is the import but the package name is `beautifulsoup4`: https://github.com/psf/requests-html/pull/296

There is also an error from pyppeteer but that should be fixed by https://github.com/conda-forge/pyppeteer-feedstock/pull/9

Can you disable the check and add a comment that `pip check` should be re-enabled when there is a new release?

File: recipes/requests-html/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - bs4
```
Comment (chrisburr) [inline recipes/requests-html/meta.yaml:23]:
```suggestion
    - beautifulsoup4
```

**Pull request merged**